js/main.js

```diff
@@ -1,11 +1,9 @@
 window.onload = function() {
+
     //start the webgazer tracker
     webgazer.setRegression('ridge') /* currently must set regression and tracker */
         .setTracker('clmtrackr')
         .setGazeListener(function(data, clock) {
-            //log the data
```

Comment:
did we write this? I didn't and if nobody else did I don't think we should remove it

Reply:
Resolved - added back in